vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/create/ProjectCreationDialog.java

```diff
@@ -151,31 +145,31 @@ private void initForm() {
     formLayout.add(defineExperimentComponent);
     formLayout.add(projectContactsLayout);
     formLayout.add(principalInvestigator);
-    formLayout.add(responsiblePerson);
     formLayout.add(projectManager);
+    formLayout.add(responsiblePerson);
     // set form layout to only have one column (for any width)
     formLayout.setResponsiveSteps(new ResponsiveStep("0", 1));
   }
 
   private void styleForm() {
-    formLayout.setClassName("create-project-form");
+    formLayout.setClassName("project-form-layout");
```

Comment:
NIT: IMO We can keep this more concise since we only select this in the css after selecting the .create-project-dialog class 
```suggestion
    formLayout.setClassName("content");
```